### ISSUE TITLE
Animation Not Playing

### ISSUE THREAD

**Issue by pluto-bell** [airbnb/lottie-web#3098]
**Tell us about your environment**

* **Latest version of Chrome**
* **After Effects Version: 2024, 24.5**
* ** Tell us about your environment**
* **Mac Sonoma 14.1**
* **Bodymovin 14.5 **
* **ZXP Installer from AEScripts**

**What did you expect to happen?**
To have the animation work/play on my page. 

**What actually happened? Please include as much relevant detail as possible.**
I rendered two separate instances. In one, the middle object shows, in the the other it does not. 
Neight rendering is animated even though autoplay and loop have a value of true. 

**Please provide a download link to the After Effects file that demonstrates the problem.**
[bodymovinIssue.zip](https://github.com/user-attachments/files/16171595/bodymovinIssue.zip)

**Comment by muuvmuuv** [comment]
I am not event seeing my animations, what is your code?

**Comment by pluto-bell** [comment]
> I am not event seeing my animations, what is your code?

My code + files are in the zip file included on original post.

**Comment by muuvmuuv** [comment]
Yeah latest realease seems broken on my installation, I just roll back to 0.28.0 and it works again.
